### PR TITLE
edits to complexity section

### PR DESCRIPTION
**Pull request by petrelharp**
Here's a start; I'll now go through again and put in more detail.  ping #66

### CODE REVIEW

**Comment by petrelharp** [comment]
OK, I've added more talking through things, and another equation.  I think it could be better, but it's probably good enough - pending your approval or edits.

**Comment by petrelharp** [comment]
merging this so we don't have competing PRs but feel free to discuss here anyhow

File: forwards_paper.tex [hunk 1]
```diff
@@ -823,27 +828,38 @@ \subsubsection*{Estimates of run-time complexity}
...
 (except for changes affecting the root, which require fewer; see \citet{kelleher2016efficient}).
 Coalescent theory tells us that
-the expected total length of the edges of a marginal tree is approximately $4N\log(2N)$,
-which is also equal to the mean number of ancestral recombination events that occur on a branch of the marginal tree,
-since we have taken one crossover per individual per generation.
+the expected total length of the edges of a marginal tree is approximately $4N\log(2N)$.
+Since we have taken one crossover per individual per generation (as is typical for a single chromosome),
+and we are measuring tree lengths in units of generations,
+the mean number of ancestral recombination events that occur somewhere on the marginal trees
+per unit of genetic length.
```
Comment (jeromekelleher) [inline forwards_paper.tex:840]: Something's missing in this sentence.

Comment (petrelharp) [reply forwards_paper.tex:840]: curses, yes.

**Pull request merged**